README.md

```diff
@@ -1,4 +1,73 @@
 # react-tfjs-models
 
-A set of components and utilities to create machine learning applications using
-React.
+`react-tfjs-models` is a set of components and utilities to create machine learning applications using
+React. It's based on Google's [tensorflow tfjs models](https://github.com/tensorflow/tfjs-models), including 
+image classification, pose detection, face detection, body segmentation and more.
+
+Comparing to integrating the underlying library, this project provides various supports for non-machine learning
+experts to use these models in their Rect applications.
+
+## React components hierarchy
+
+`react-tfjs-models` has provided a more intuitive declarative syntax, rather than the traditional imperative 
+approach. An application to use `BlazePose` model to analyze each frame from a webcam stream would look like:
+
+```
```

Comment:
[Declare the language](https://github.com/google/styleguide/blob/gh-pages/docguide/style.md#declare-the-language):

> It is best practice to explicitly declare the language, so that neither the syntax highlighter nor the next editor must guess.
```suggestion
```jsx
```